backupctl.spec

```diff
@@ -0,0 +1,47 @@
+%define __python /usr/bin/python3
+
+Name:           backupctl
+Version:        1.1.2
+Release:        1%{?dist}
+Summary:        Tool to manage zfs volumes and create new dirvish vault configurations.
+License:        GPLv3
+
+Source0:        https://github.com/adfinis-sygroup/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python34 python34-setuptools python34-docutils
+BuildRoot:      %{_tmppath}/%{name}-%{version}
+
+Requires:       python34-jinja2, python34-setuptools, python34-pyxdg, dirvish, zfs
+
+%description
+Tool to manage zfs volumes and create new dirvish vault configurations.
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+%{__python} setup.py build
+rst2man-3.4 backupctl.8.rst > backupctl.8
+rst2man-3.4 backupctl.ini.5.rst > backupctl.ini.5
+gzip -c backupctl.8 > backupctl.8.gz
+gzip -c backupctl.ini.5 > backupctl.ini.5.gz
+
+%install
+rm -rf %{buildroot}
```

Comment:
AFAIK this is now obsolete and it's not recommended to have it in all the scriptlets anymore.